lambdas/initialise-ipv-session/src/test/java/uk/gov/di/ipv/core/initialiseipvsession/InitialiseIpvSessionHandlerTest.java

```diff
@@ -821,67 +821,6 @@ void shouldNotStoreInheritedIdentityWhenVotWeakerThanExisting() throws Exception
             verify(mockVerifiableCredentialService, times(0)).persistUserCredentials(any());
         }
 
-        @Test
```

Comment:
Did Sonar request this test deletion?

Reply:
It was complaining that the test body was identical as another. So it made sense to delete it as its duplicated test which just the name slightly varying

Reply:
Oh, nice :)